src/Components/Activities/Activities.css

```diff
@@ -1,39 +1,32 @@
-body, html {
-  height: 100%;
-  width: 100%;
-  background: rgb(238,174,202);
-  background: radial-gradient(circle, rgba(238,174,202,1) 0%, rgba(148,187,233,1) 100%);
+html {
+  font-family: var(--font-family);
 }
 
 h2 {
   margin: 20px;
 }
 
-button {
+.activities-btns {
   font-size: 30px;
   border: none;
   background-color: white;
-}
-
-.go-button {
-  font-size: 19px;
-  padding: 5px;
-  background-color: black;
-  color: white;
-  font-family: 'Playfair Display', serif;
-  font-weight: bold;
+  border-radius: 2px;
+  cursor: pointer;
 }
 
 .activity-card {
   display: flex;
   justify-content: center;
   align-items: center;
+  height: 300px;
+  width: 40%;
   margin-top: 10%;
   margin-bottom: 2%;
   padding: 50px;
   font-size: 23px;
   background-color: white;
```

Comment:
would love to see some opacity added here so it's a little softer ? (also on the journal prompt page?)

Reply:
I love that the activity cards stay the same size! 

One thought for the future: I'd love to update the responsiveness on this page at some point and maybe make the text shrink with window size? And maybe move the arrows below the text at a certain window size?